.storybook/pages/Cad/Cad.tsx

```diff
@@ -16,208 +16,591 @@ import {
+  });
+  const [items, setItems] = useState({
+    'item-1': {
+      title: 'Project #1',
+      children: (
+        <ProjectCard
+          behavior="draggable"
+          buttonDropdownItems={
+            <>
+              <DropdownMenuItem>
+                <Icon name="schedule" purpose="decorative" size="1.25rem" />
+                Move to other section
+              </DropdownMenuItem>
+              <DropdownMenuItem>
+                <Icon name="schedule" purpose="decorative" size="1.25rem" />
+                Move up
+              </DropdownMenuItem>
+              <DropdownMenuItem>
+                <Icon name="schedule" purpose="decorative" size="1.25rem" />
+                Move down
+              </DropdownMenuItem>
+              <DropdownMenuItem status="error">
+                <Icon name="delete" purpose="decorative" size="1.25rem" />
+                Delete item
+              </DropdownMenuItem>
+            </>
           }
-          title="Course Planner: History 6"
-        />
-      </LayoutContainer>
-    </Panel>
-    <LayoutContainer>
-      <Layout sidebar="wide" variant="right-sidebar">
-        <LayoutSection region="main">
-          <Panel>
-            <Heading as="h2" className="u-margin-bottom-lg" size="headline-sm">
-              Select projects for your History 6 plan
-            </Heading>
-            <Text as="div" className="u-margin-bottom-lg">
-              <p>
-                Make a plan so that you can stay connected to learning
-                objectives even as changes occur throughout the year.
-              </p>
-              <p>
-                Room for more instructional copy per Content Strategy lorem
-                ipsum dolor sit amet, consectetur adipiscing elit. Phasellus est
-                quam, consequat iaculis pretium accumsan, fringilla id ligula.
-              </p>
-            </Text>
+          meta="12 days"
+          number={1}
+          numberAriaLabel="Project 1"
+          title="Longer project card title that wraps"
+        ></ProjectCard>
+      ),
+    },
+    'item-2': {
+      title: 'Project #2',
+      children: (
+        <ProjectCard
+          behavior="draggable"
+          buttonDropdownItems={
+            <>
+              <DropdownMenuItem>
+                <Icon name="schedule" purpose="decorative" size="1.25rem" />
+                Move to other section
+              </DropdownMenuItem>
+              <DropdownMenuItem>
+                <Icon name="schedule" purpose="decorative" size="1.25rem" />
+                Move up
+              </DropdownMenuItem>
+              <DropdownMenuItem>
+                <Icon name="schedule" purpose="decorative" size="1.25rem" />
+                Move down
+              </DropdownMenuItem>
+              <DropdownMenuItem>
+                <Icon name="schedule" purpose="decorative" size="1.25rem" />
+                Move view details
+              </DropdownMenuItem>
+            </>
+          }
```

Comment:
Is this `buttonDropdownItems` the same as the ones below? Might be nice to collapse those into one variable to cut the file down a little.

Reply:
I can do that. These pages are a little less critical (at least to me) from a composition standpoint simply because product teams will take this as a starting point and roll with it